Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM registry.redhat.io/ubi8/go-toolset:latest as builder
+FROM registry.redhat.io/ubi8/go-toolset:1.19 as builder
 WORKDIR /go/src/app
 COPY . .
 USER 0
 RUN go get -d ./... && \
-    go build -o rosocp rosocp.go
+    go build -o rosocp rosocp.go && \
+    echo "$(go version)" > go_version_details
```

Comment:
I don't think we'll need to do this if specific tag is mentioned for go-toolset

Reply:
We could pin the patch/subversion tag as well

Reply:
Yes, we can, however that can be overkill considering we would end up not getting the fixes by default in patched versions.